src/mixins/helpers.js

```diff
@@ -8,9 +9,9 @@ import assign from 'object-assign';
 var helpers = {
   initialize: function (props) {
     var slideCount = React.Children.count(props.children);
-    var listWidth = this.getWidth(this.refs.list.getDOMNode());
-    var trackWidth = this.getWidth(this.refs.track.getDOMNode());
-    var slideWidth = this.getWidth(this.getDOMNode())/props.slidesToShow;
+    var listWidth = this.getWidth(this.refs.list);
+    var trackWidth = this.getWidth(ReactDom.findDOMNode(this.refs.track));
```

Comment:
You can use directly `this.refs.tracks`. Because DOMNode is already bound to refs (see: https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#dom-node-refs)